src/Http/Http.Abstractions/test/Validation/ValidateContextTests.cs

```diff
@@ -0,0 +1,207 @@
+#pragma warning disable ASP0029 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
Let's remove this test file. It's not providing any value considering the other test file we have.